tests/assets/default.conf

```diff
@@ -13,9 +13,8 @@ databases 16
 latency-monitor-threshold 1
 repl-diskless-sync-delay 0
 
+# Note the infrastructure in server.tcl uses a dict, we can't provide several save directives
 save 900 1
-save 300 10
-save 60 10000
```

Comment:
In any case this is a deprecated format, why not use:
`save 900 1 300 10 60 10000`
in _default.confg_?

Reply:
What I did was in order to have the same outcome as what we had so far. 
And decided to add a comment so that others won't try to add more `save` lines.